documentation/setup.md

```diff
@@ -57,4 +57,4 @@ The official releases and previews are:
 - Documented on the [Releases](./releases.md) page.
 - Announced on the GitHub [Releases](https://github.com/dotnet/dotnet-monitor/releases) tab.
 - Published to [nuget.org](https://www.nuget.org/packages/dotnet-monitor/).
-- Published to the Microsoft Container Registry ([tag list](https://mcr.microsoft.com/v2/dotnet/monitor/tags/list)).
\ No newline at end of file
+- Published to the Microsoft Container Registry ([tag list](https://mcr.microsoft.com/v2/dotnet/monitor/tags/list)).
```

Comment:
What changed here?

Reply:
It may have added a newline; there should be no change here.